### ISSUE TITLE
Latency and response time

### ISSUE THREAD

**Issue by KinaneD** [postaddictme/instagram-php-scraper#596]
We have been using this library for a while on production. We have observed that response time has increased drastically.

To get an account details it take ~11s! that is using `Instagram::getAccount()` and sure enough using `Instagram::getAccountById()` takes almost double that time.

In any real-time applications, these numbers are not feasible at all.

Is anyone else experiencing similar behaviour?
Any possible performance improvement in this regard?

Thank you in advance.

**Comment by jangrewe** [comment]
Yes. I guess it's IG tarpitting your IP/account. Cease any activity for a couple of hours/days, and it should get faster again. YMMV.

**Comment by KinaneD** [comment]
Sure.
We previously went with a proxy service to work around Instagram's greed. First we used datacenter proxies, as expected some of the requests fail. Then we moved to residential proxies where all request goes through.

We hoped this will do the trick but here we are.

Is there any further tempering with could do?
For example i can think of using different user agent per request.

Thoughts?
Thank you.
